client/lib/post-normalizer/utils.js

```diff
@@ -30,6 +30,16 @@ export function imageSizeFromAttachments( post, imageUrl ) {
 	}
 }
 
+export function stripPhotonParams( imageUrl ) {
+	if ( ! imageUrl ) {
+		return imageUrl;
+	}
+	const parsedUrl = url.parse( imageUrl );
+	parsedUrl.query = parsedUrl.search = ''; // remove query params
```

Comment:
we should try to keep any non-sizing type params. From what I've seen, they're quite rare in the wild, but they would affect the image in a rather unexpected way